analyzere/__init__.py

```diff
@@ -2,6 +2,7 @@
 password = ''
 base_url = 'http://localhost:8000/'
 upload_poll_interval = 0.1
+upload_chunk_size = 16 * (1 << 20)
```

Comment:
better I think would be either the 1k * 1k version or the actual thing if we actually want to do 2^20.  In python I think it's `2**20` 
we all know left shift is the same as raising 2 to a power, but it's not quite the first think you think of when talking about an absolute size of something.  bit operations usually make the mind go elsewhere.